libraries/common/effekt.effekt

```diff
@@ -42,11 +42,19 @@ extern def println(value: String): Unit =
   js "$effekt.println(${value})"
   chez "(println_impl ${value})"
   llvm """
-    call void @c_io_println_String(%Pos ${value})
+    call void @c_io_println(%Pos ${value})
     ret %Pos zeroinitializer ; Unit
   """
   vm "effekt::println(String)"
 
+extern async def readln(): String =
+  js "$effekt.capture(callback => $effekt.readln(callback))"
```

Comment:
Maybe use `jsNode`, since this doesn't work on the web?
(You could then also have a separate `jsWeb` implementation)

Alternatively, we can check in which case we are in `$effekt.readln` and act accordingly.

Reply:
i.e.
```suggestion
  jsNode "$effekt.capture(callback => $effekt.readln(callback))"
```

Reply:
In addition to that, we could also put `$effekt.readln` into an `extern jsNode """ .... """` block directly in `effekt.effekt` (above this defn)